### ISSUE TITLE
Allow adding Changelog

### ISSUE THREAD

**Issue by rugk** [wdzeng/firefox-addon#2]
For a release, on AMO, you can manually upload a changelog (with a limited set of HTML), in different languages.

I e.g. maintain that data manually in `.html` files, like here: https://github.com/rugk/offline-qr-code/tree/master/assets/texts/en/Changelog

Maybe you can just allow a file path as input, which is then used as a changelog. (Maybe also a whole language directory, given how else may one automate that for all languages?)

**Comment by wdzeng** [comment]
Hi @rugk, did you mean the "Version Notes"?

![image](https://github.com/wdzeng/firefox-addon/assets/39057640/f9f3b528-0af1-40f2-97ce-71529913f0aa)

**Comment by rugk** [comment]
Yes that's what I mean… it's kinda an oldish term :wink:

**Comment by wdzeng** [comment]
Hi @rugk, did you want to try v1.1.0-alpha.0?

```yaml
steps:
  - uses: wdzeng/firefox-addon@v1.1.0-alpha.0
    with:
      addon-guid: your-addon's-guid
      xpi-path: your-addon.zip
      self-hosted: false
      release-notes: {"en-US":"foo"}
      jwt-issuer: ${{ secrets.FIREFOX_JWT_ISSUER }}
      jwt-secret: ${{ secrets.FIREFOX_JWT_SECRET }}
```

The `release-notes` accepts a JSON string. You will have to try to retrieve such value in the previous steps in the workflow. Thanks!

**Comment by rugk** [comment]
Hmm tried a draft, but first would need to find out how to read and all files in all localizations available… hmmpff… https://github.com/jaywcjlove/github-action-read-file/issues/12

Edit: Tried something in https://github.com/rugk/mastodon-simplified-federation/pull/127

**Comment by wdzeng** [comment]
Landed on v1.1.
